pkg/provider/github/parse_payload.go

```diff
@@ -395,8 +395,8 @@ func (v *Provider) handleCommitCommentEvent(ctx context.Context, event *github.C
 	runevent.SHA = event.GetComment().GetCommitID()
 	runevent.HeadURL = runevent.URL
 	runevent.BaseURL = runevent.HeadURL
-	runevent.EventType = "push"
-	runevent.TriggerTarget = "push"
+	runevent.TriggerTarget = triggertype.Push
+	opscomments.SetEventTypeAndTargetPR(runevent, event.GetComment().GetBody())
```

Comment:
I tried to verify 
`/hello-world` works fine on latest pushed commits
but when i try to do on different branches 

```
/hello-world branch:newbranch
```
Its not working as expected where as `/test branch:newbranch` works fine

Reply:
it's fine i guess to say that's not working properly since my feeling is that it's a bit of a edge case